types/aws-lambda/trigger/kinesis-stream.d.ts

```diff
@@ -28,7 +31,20 @@ export interface KinesisStreamEvent {
     Records: KinesisStreamRecord[];
 }
 
-// https://docs.aws.amazon.com/lambda/latest/dg/with-kinesis.html
+// https://docs.aws.amazon.com/lambda/latest/dg/with-kinesis.html#services-kinesis-windows
+export interface KinesisStreamTumblingWindowEvent {
+    window: { start: string; end: string };
+    state?: { [key: string]: any };
+    isFinalInvokeForWindow: boolean;
+    isWindowTerminatedEarly: boolean;
+    Records: KinesisStreamRecord[];
```

Comment:
Remove `Records` as `KinesisStreamEvent` provides it.

Reply:
Done so